apps/cowswap-frontend/src/modules/tradeQuote/hooks/useQuoteParams.ts

```diff
@@ -45,7 +45,7 @@ export function useQuoteParams(amount: string | null): LegacyFeeQuoteParams | un
       toDecimals,
       fromDecimals,
       isEthFlow: false,
-      priceQuality: getPriceQuality({ verifyQuote: enoughBalance && verifiedQuotesEnabled }),
+      priceQuality: getPriceQuality({ verifyQuote: verifiedQuotesEnabled && enoughBalance === true }),
```

Comment:
Do you need to compare it with true?
Would double negation be cleaner?

```suggestion
      priceQuality: getPriceQuality({ verifyQuote: verifiedQuotesEnabled && !!enoughBalance }),
```

Reply:
Never mind, code was updated

Reply:
In the last commit i just changed the signature of getPriceQuality to accept `undefined` 

There was a bit if repetition of this logic, so i just made that function to handle the UNKNOWN state.

Reply:
I see you check the code before my last commit (where i removed all those === true), just to handle it inside `getPriceQuality` c56eb65324c720e6d272ac10a92456bd16450eee